.github/workflows/publish.yml

```diff
@@ -19,8 +19,8 @@ jobs:
       - name: Upgrade pip
         run: pip install --upgrade pip
 
-      - name: Install build & pdm-backend
-        run: pip install build pdm-backend
+      - name: Install build, pdm-backend, and grpcio-tools
+        run: pip install build pdm-backend grpcio-tools
```

Comment:
Here we need to specify exact version of `grpcio-tools` package in order to avoid version compatibility errors at runtime.
Current script will be installing the latest one (I guess it's `1.70.x`), but the SDK actually uses `1.68.1` ([link](https://github.com/hiero-ledger/hiero-sdk-python/blob/main/pyproject.toml#L17)). So we probably want to stick with `1.68.1`, the other option is to update the version in SDK dependencies.

Example error:
```
E   google.protobuf.runtime_version.VersionError: Detected incompatible Protobuf Gencode/Runtime versions when loading query.proto: gencode 5.29.0 runtime 5.28.1.
```